source/hardware/controllers/denon_mc7000.rst

```diff
@@ -301,7 +304,8 @@ This mode lets you jump a number of beats while pushing a pad button once.
 Sampler Mode (pink LED)
 -----------------------
 
-8 samplers can be triggered from either Deck.
+8 samplers can be triggered from each Deck if MC7000.SamplerQty is set to 32. Deck 1 will play sampler 1 to 8, Deck 2 will play sampler 9 to 16, Deck 3 will play sampler 17 to 24 and Deck 4 will play sampler 25 to 32.
```

Comment:
Most users will simply use the default mapping without any customization. The manual should primarily describe this and simply refer to the script if users intend to customize it.

As a regular user without any ambitions to program JavaScript I find the references to internals of the script more confusing than helpful.

TL;DR IMHO it would be sufficient to describe only the default behavior in the manual and document the options for customization only in the script code.

Reply:
I set the default to 16 samplers in the script now, so the user does not need to take any action before using, this works with the default skin.

Reply:
comment for 32 samplers is now in the script code.